spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/restart/RestartApplicationListener.java

```diff
@@ -67,6 +72,7 @@ private void onApplicationStartingEvent(ApplicationStartingEvent event) {
 			Restarter.initialize(args, false, initializer, restartOnInitialize);
 		}
 		else {
+			logger.info("RestartApplicationListener is disabled.");
```

Comment:
Logging here could be useful, particularly if it indicated when the restarter hasn't been initialised, but it doesn't meet the requirement described in #14492. That issue is specifically interested in the case where `restartOnInitialize` is false because an agent reloader is active.

Reply:
Mhm maybe I gonna close this one and bring it together if I find a resolution to satisfy #14492 then.